plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -178,12 +178,8 @@ func (s *serviceAccount) Admit(a admission.Attributes) (err error) {
 
 	// Ensure the referenced service account exists
 	serviceAccount, err := s.getServiceAccount(a.GetNamespace(), pod.Spec.ServiceAccountName)
-	if err != nil {
-		return admission.NewForbidden(a, fmt.Errorf("error looking up service account %s/%s: %v", a.GetNamespace(), pod.Spec.ServiceAccountName, err))
-	}
-	if serviceAccount == nil {
-		// TODO: convert to a ServerTimeout error (or other error that sends a Retry-After header)
```

Comment:
Hold on https://github.com/kubernetes/kubernetes/pull/55252 .  @liggitt you still want to do it?  I remember this as being your comment, but I haven't gotten a lot of complaints as-is.

Reply:
I probably wouldn't at this point

Reply:
> I probably wouldn't at this point

Mirrors my thoughts.  I removed the comment in my pull.

Reply:
Should we close this pr?